docs/development.rst

```diff
@@ -64,6 +64,15 @@ will override the defaults.
 Server Development
 ------------------
 
+All commits to the core python code must work in all versions of python 2.7 and above.
+Python code in plugins should also work in 2.7 and 3.x, but some plugins may depend
```

Comment:
Do we want to claim 3.x?  I think we just want to say 3.4 and above, otherwise we are committing to testing with 3.0, etc.